### DISCUSSION TITLE
Mobile Supprt PLZ

### DISCUSSION THREAD

**Discussion by newsgood** [gavvvr/obsidian-imgur-plugin#55] [Ideas]
I love your plugin but unhappy not to be able to use it in my mobile environment, especially in SAMSUNG Dex mode.

You may know that Dex mode with BIg screens is almost like PC environment.

Thanks in advance!

**Comment by gavvvr** [comment]
Hi @newsgood

It was requested long time ago (#13), so it duplicates an old feature request :)
Since it was initially requested I always wanted the plugin to work on mobile, but there were some difficulties to debug and refactor the code to enable mobile support.

> ... especially in SAMSUNG Dex mode.
> You may know that Dex mode with BIg screens is almost like PC environment.

Never thought about such Obsidian mobile app use-case (as a desktop app in DEX mode), but this argument makes mobile support feature request even more important.

Today I am happy to announce that there is a v2.4.0 released just now, which supports mobile devices. Check it out :)

**Comment by newsgood** [comment]
Nothing but 👍👍👍

Sent by 류형석 (Ray Ryu) / Mobile +82-10-9918-xxxx

2023년 10월 20일 (금) 22:19, Kirill Gavrilov ***@***.***>님이 작성:
